src/book/03-guides/03-hardware/01-working-with-robots.mdx

```diff
@@ -5,3 +5,62 @@ title: Working with Robots
+    - Remove the arm from the shoulder socket, it will still be attatched by a wire
+    - Take off the long part of plastic between the shoudler and neck, it is held by two screws. This piece of plastic can be seen in the following photo:
+    ![shoulder cover](./images/shoulder/shoulder.jpg 'shoulder cover')
+    - Now that the piece is off, there is a port for the chord that is still keeping the arm in place, detach this and the arm will be completely detatched
+2. Detatching the shoulder
+    - Looking in the shoulder socket there is another circle of screws, remove these and the shoulder will detatch
+    Note: Two plastic spacers should also fall out
+3. Removing the servo
+    - Remove and disconnect the boards and fans neccessary to access the shoulder servo from the inside, make sure to note any chords you detatch
+    - On the outside of the robot where the shoulder socket detatched, remove the four screws mounting the servo to the robots body
+    - There is a plastic holder inside the robot holding each shoulder servos in place, slide this towards the robots back to unclip the servos to allow it to fall freely out of its slot
+
+4. Mechanically testing the servo
+    - Remove the circular metal plate from the servo, get the plastic servo testor to test if the servo rotates freely without any clunks. If it doesn't rotate or is clunky, it needs replacing
+5. Installing a new servo
+    - Get the new servo set with the same serial number on the box
+    - Connect the metal circular disk that came in the box on the new servo, ensuring you line up the indents on the metal disk and servo
+    - Place the servo into the body, put back the plastic holder and reconnect all the chords that were disconnected in the 'removing servo' step
+    - Rescrew the four screws that were taken out 
+    - The arms of the robot should be directly outwards when set to 0 degrees. This means you need to orientate the server before placing the shoulder back on. 
+    On the metal circular disk that was placed on, there are small indents in the metal, two on one side and one on the other.
+    The single indent needs to be towards the front of the robot, and the two dots towards its back, this ensures the robots shoulder will be correctly aligned.
+
+6. Reconstruction 
+    - Now attach the shoulder back on, ensuring you put back the plastic spacers and leave one hole without a screw (the biggest hole in the circle of screws). The shoulder should also be placed on
+    with correct orientation. On the shoulder there is one side of raised up plastic, two sides with the holes for the screws and a shorter side. The side with the shortest bit of plastic should be faced frontwards,
+    the same direction the metal circular disk was orientated.
+    - Connect the chord of the arm again to the slot between the shoulder and head
+    - slide the arm in place and rescrew the arm to the shoulder
```

Comment:
The arrow on the servo horn also needs to point in a very specific direction here as well

Reply:
The metal circular disk is the horn, I thought new recruits might not know its called a horn. Should I put that in?

Reply:
Maybe at the start of this section, include a part on they different servos (mx64 mx106) and what a flywheel and horn are

Reply:
We should use correct terms wherever possible. You are right though, new recruits are unlikely to know these terms. We will have to make another page (or add to this page) details about the servos (might make more sense on the page where we detail how to change the gear set in the servo).

To be more specific
-  The horn is more of a bronze-ish colour. It has the three dots that form the arrow on it. It won't spin freely by hand (you will almost always need to use an allen/hex key as a lever to make it spin)
- The flywheel sits on the opposite side of the servo (not all servo in the robot have a flywheel attahced). The flywheel is silver and it free-spinning (you can flick it with you finger and it will spin happily)

Reply:
Good idea, I will add that section and get some photos as well